core/src/main/resources/lib/layout/expandButton.jelly

```diff
@@ -1,42 +0,0 @@
-<!--
```

Comment:
1 plugin uses this per https://github.com/search?q=org%3Ajenkinsci+expandbutton&type=Code and it's suspended: https://plugins.jenkins.io/javanet/

GitHub code search is known incomplete though, did you check all plugins' resources? This also is a breaking change that needs to be documented.

Reply:
Nothing extra in the new code search which is a lot better: https://cs.github.com/?scopeName=All+repos&scope=&q=org%3Ajenkinsci+expandButton